### ISSUE TITLE
Opening snap on cardiac auscultation

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#9935]
**Preferred term label:**
Opening snap on cardiac auscultation

**Synonyms**

**Definition (free text, please give PubMed ID)**
https://www.ncbi.nlm.nih.gov/medgen/C0232275

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Mitral stenosis

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000685
- label: Mitral opening snap
- parents: [Abnormal heart sound (HP:0031657)](https://hpo.jax.org/app/browse/term/HP:0031657)
- pmid: PMID:17448551
- def: Opening snap is an additional sound heard in the diastole that is related to the opening of a stenotic but mobile mitral valve. It is described as an early diastolic, high-pitched sound, which is associated with opening of the mitral and/or tricuspid valve.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/9935
- synonyms:
